### PR TITLE
Minim si maxim

### PR DESCRIPTION
**Pull request by TudorChindea**

### CODE REVIEW

File: Chindea Tudor Eugen/Lab3 CP/main.c [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+int main()
+{
+    // continue - este folosit in for  si cand se apeleaza se intoarce la inceput de for si creste contorul
+    // continue e poate folosi in orice functie repetitiva; te aduce la inceputul instructiunilor
+    int sequence[100],n,mini,pozmin[100],maxi,pozmax[100],ctmin=1,ctmax=1,i;
+    scanf("%d",&n);
+    for(i=0; i<n; i++)
+    {
+        scanf("%d",&sequence[i]);
+        if(i==0)
```
Comment (Alex-Cosma) [inline Chindea Tudor Eugen/Lab3 CP/main.c:13]: You should initialize `mini`,`maxi`,`pozmin` and `pozmax` at the beginning so you won't need to use this if here
Also, why are `pozmin` and `pozmax` vectors? it doesn't make sense

File: Chindea Tudor Eugen/Lab3 CP/main.c [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+        }
+        else
+        {
+            if(sequence[i]>maxi)
+            {
+                maxi=sequence[i];
+                ctmax=0;
+                pozmax[ctmax]=i;
+            }
+            else if(sequence[i]==maxi)
```
Comment (Alex-Cosma) [inline Chindea Tudor Eugen/Lab3 CP/main.c:28]: Well, if this is true, then the maximum number is still `maxi`, and I think you can simply leave it as it is, no need to do anything more about it

File: Chindea Tudor Eugen/Lab3 CP/main.c [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+                ctmax++;
+                pozmax[ctmax]=i;
+            }
+            if(sequence[i]<mini)
+            {
+                mini=sequence[i];
+                ctmin=0;
+                pozmin[0]=i;
+            }
+            else if(sequence[i]==mini)
```
Comment (Alex-Cosma) [inline Chindea Tudor Eugen/Lab3 CP/main.c:39]: Same for minimum, just leave as is

**Review by Alex-Cosma — CHANGES_REQUESTED** [review]

File: Chindea Tudor Eugen/Lab3-pb 3.4/main.c [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+    int diviz=2;
+    while(diviz*diviz<=x)
+    {
+        if(x%diviz==0)
+            return 0;
+        diviz++;
+    }
+    return 1;
+}
+int main()
```
Comment (Alex-Cosma) [inline Chindea Tudor Eugen/Lab3-pb 3.4/main.c:14]: Goodie

**Review by Alex-Cosma — CHANGES_REQUESTED** [review]

**Comment by TudorChindea** [comment]
pozmin and pozmax are vectors because the problem requires to print the positions.. Not the first position where the maximum and minimum walues appear

Sent from Yahoo Mail on Android 
 
  On Wed, Oct 16, 2019 at 13:12, Cosma Alex<notifications@github.com> wrote:   
@Alex-Cosma requested changes on this pull request.

In Chindea Tudor Eugen/Lab3-pb 3.4/main.c:
> @@ -0,0 +1,27 @@
+#include <stdio.h>
+#include <stdlib.h>
+int nrprim(int x)
+{
+    int diviz=2;
+    while(diviz*diviz<=x)
+    {
+        if(x%diviz==0)
+            return 0;
+        diviz++;
+    }
+    return 1;
+}
+int main()

Goodie

File: Chindea Tudor Eugen/Lab3 CP/main.c (hunk 1 above)
Comment (Alex-Cosma) [reply Chindea Tudor Eugen/Lab3 CP/main.c:13]: Fair enough, you are right on that

**Pull request merged**